collect_app/src/main/res/layout/counter_widget.xml

```diff
@@ -0,0 +1,41 @@
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <ImageView
+        android:id="@+id/minus_button"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:src="@drawable/ic_baseline_remove_24"
+        android:background="@drawable/counter_minus_button_background"
+        android:paddingVertical="@dimen/margin_small"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toStartOf="@+id/value" />
+
+    <TextView
+        android:id="@+id/value"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:gravity="center"
+        android:background="@drawable/counter_value_background"
+        android:textSize="25sp"
```

Comment:
@grzesiek2010 @alyblenkin let's work out a Material 3 text appearance for this to use instead of a custom text size. If there isn't one that feels right, we can tweak one of the appearances to suit us better.

Use disparate text sizes like this will cause us problems down the line when we change themes.

Reply:
Sounds good.

Reply:
`textAppearanceHeadlineSmall` looks nice.